kproject-common/src/main/kotlin/com/soywiz/kproject/internal/Yaml.kt

```diff
@@ -42,7 +42,10 @@ internal object Yaml {
                     list.add(res)
                 } else {
                     if (TRACE) println("${levelStr}CHILD.return: $res")
-                    return res
+                    (res as MutableMap<String, Any?>).forEach { (key, value) ->
+                        if (map != null) { map!![key] = value }
+                    }
+                    return map
```

Comment:
Can you add a test (that failed before and works now) for this so I can understand what this is doing?

https://github.com/korlibs/kproject/blob/main/kproject-common/src/test/kotlin/com/soywiz/kproject/YamlTest.kt

Reply:
I am checking. I looked in the unit tests and realized that the "real" problem is somewhere else. So this change is not needed. I will update you since now I need some care time :)